tools/compliance/write_sbom.py

```diff
@@ -67,21 +70,40 @@ def create_sbom(package_info: dict) -> dict:  # pylint: disable=g-bare-generic
   })
 
   for pkg in package_info["packages"]:
-    packages.append(
-        {
-            "name": pkg,
-            # TODO(aiuto): Fill in the rest
-            # "SPDXID": "SPDXRef-GooglePackage-4c7dc29872b9c418",
-            # "supplier": "Organization: Google LLC",
-            # "downloadLocation": "NOASSERTION",
-            # "licenseConcluded": "License-da09db95a268defe",
-            # "copyrightText": ""
-        }
-    )
+    id = hashlib.md5()
+    id.update(pkg.encode('utf-8'))
+    spdxid = 'SPDXRef-GooglePackage-%s' % id.hexdigest()
+    pi = {
+        "name": pkg,
+        "downloadLocation": "NOASSERTION",
+        "SPDXID": spdxid,
+        # TODO(aiuto): Fill in the rest
+        # "supplier": "Organization: Google LLC",
+        # "licenseConcluded": "License-da09db95a268defe",
```

Comment:
minor nit:
if the License line is fake, I think should replace with License-###########

mostly to make it immediately clear that these lines are example placeholders versus arbitrarily commented out.

Reply:
Sure. Done